packages/cashier/src/Components/recent-transaction.jsx

```diff
@@ -0,0 +1,114 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Button, Text, Icon } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import { epochToMoment } from '@deriv/shared';
+import { connect } from 'Stores/connect';
+
+const RecentTransaction = ({ crypto_transactions, currency }) => {
+    const getStatus = status => {
+        switch (status) {
+            case 'CONFIRMED':
+                return 'Successful';
+            case 'LOCKED':
+                return 'In review';
+            case 'PERFORMING_BLOCKCHAIN_TXN':
+                return 'In process';
+            case 'PROCESSING':
+                return 'In process';
+            case 'SENT':
+                return 'Successful';
+            case 'VERIFIED':
+                return 'In process';
+            default:
+                return status[0].toUpperCase() + status.slice(1).toLowerCase();
+        }
+    };
+
+    let { address_hash, status_code, submit_date, transaction_hash, transaction_type } = crypto_transactions[
+        crypto_transactions.length - 1
+    ];
+
+    submit_date = epochToMoment(submit_date).format('MMM D, YYYY');
+    transaction_type = transaction_type[0].toUpperCase() + transaction_type.slice(1);
+    transaction_hash = transaction_hash
+        ? `${transaction_hash?.substring(0, 4)}....${transaction_hash?.substring(transaction_hash.length - 4)}`
+        : 'Pending';
+    address_hash = `${address_hash.substring(0, 4)}....${address_hash.substring(address_hash.length - 4)}`;
+    status_code = getStatus(status_code);
+
+    const amount = crypto_transactions[crypto_transactions.length - 1].amount;
+
+    return (
+        <div className='cashier-recent-transaction-wrapper'>
+            <div className='cashier-recent-transaction'>
+                <Text weight='bold' as='p' line_height='s' size='xs'>
+                    <Localize i18n_default_text='Recent Transactions' />
+                </Text>
+                <div className='cashier-recent-transaction__data-wrapper'>
+                    <Icon
+                        className='cashier-recent-transaction__icon'
+                        icon={transaction_type === 'deposit' ? 'IcCashierAdd' : 'IcCashierMinus'}
+                        size={32}
+                    />
+                    <div>
+                        <div className='cashier-recent-transaction__status-wrapper'>
+                            <Text weight='normal' as='p' size='xxs'>
```

Comment:
Nit but `weight` is `normal` by default so you don't have to specify it but also not wrong. :)

Reply:
Alright, I'll remove it ;)